server/views/partials/prisonerBanner.njk

```diff
@@ -1,7 +1,7 @@
 <div class="app-mini-profile-header govuk-!-margin-bottom-6 govuk-!-display-none-print">
   <dl>
     <dt class="govuk-heading-s govuk-!-margin-bottom-0">
-      <a href="{{ dpsUrl }}/prisoner/{{ prisonerSummary.prisonNumber }}" class="govuk-link govuk-!-font-weight-bold" aria-label="{{ prisonerSummary.lastName | title }}, {{ prisonerSummary.firstName | title }}'s prisoner profile">
+      <a href="{{ dpsUrl }}/prisoner/{{ prisonerSummary.prisonNumber }}" class="govuk-link govuk-!-font-weight-bold"  data-id="prisoner-banner-prisoner-{{ prisonerSummary.prisonNumber }}" aria-label="{{ prisonerSummary.lastName | title }}, {{ prisonerSummary.firstName | title }}'s prisoner profile">
```

Comment:
I think we also have some breadcrumb links that might need the same treatment 🤔 
Try searching the `views` folder for the phrase:
```
{{ prisonerSummary.firstName }} {{ prisonerSummary.lastName }}'s learning and work progress
```
I think there might be 4 of them, but you might find some others (or variations of it)